test/scenarios/fork/ynETHWithdrawals.t.sol

```diff
@@ -0,0 +1,544 @@
+    error ERC721NonexistentToken(uint256 tokenId);
+    error InsufficientBalance(uint256 currentBalance, uint256 requestedBalance);
+    error InsufficientAssetBalance(address asset, uint256 requestedAmount, uint256 balance);
+    error AccessControlUnauthorizedAccount(address account, bytes32 neededRole);
+    error AmountExceedsSurplus(uint256 requestedAmount, uint256 availableSurplus);
+
+    bool public isHolesky;
+
+    address public constant user = address(0x12345678);
+    address public constant receiver = address(0x987654321);
+
+    uint256 public nodeId = 2;
+    uint256 public withdrawalAmount = 32 ether;
+
+    uint256 tokenId;
+    uint256 secondsToFinalization;
+    uint256 amountOut;
+
+    StakingNode public stakingNode;
+
+    // ------------------------------------------
+    // Setup
+    // ------------------------------------------
+
+    function setUp() public override {
+        super.setUp();
+
+        isHolesky = block.chainid == 17000;
+
+        if(isHolesky) {
+            stakingNode = StakingNode(payable(address(stakingNodesManager.nodes(nodeId))));
+
+            vm.deal(user, 10_000 ether);
+
+            setupForVerifyWithdrawalCredentials(nodeId, "test/data/holesky_wc_proof_1916455.json");
+
+            secondsToFinalization = 1 days;
+            vm.prank(actors.ops.WITHDRAWAL_MANAGER);
+            ynETHWithdrawalQueueManager.setSecondsToFinalization(secondsToFinalization);
+        }
+    }
+
+    // ------------------------------------------
+    // Withdrawal flow tests
+    // ------------------------------------------
+
+    function testVerifyWithdrawalCredentials() public {
+        if (!isHolesky) return;
+
+        ValidatorProofs memory _validatorProofs = getWithdrawalCredentialParams();
+
+        vm.expectRevert(bytes4(keccak256("NotStakingNodesOperator()")));
+        stakingNode.verifyWithdrawalCredentials(
+            uint64(block.timestamp),
+            _validatorProofs.stateRootProof,
+            _validatorProofs.validatorIndices,
+            _validatorProofs.withdrawalCredentialProofs,
+            _validatorProofs.validatorFields
+        );
+
+        uint256 _unverifiedStakedETH = stakingNode.unverifiedStakedETH();
+
+        vm.prank(actors.ops.STAKING_NODES_OPERATOR);
+        stakingNode.verifyWithdrawalCredentials(
+            uint64(block.timestamp),
+            _validatorProofs.stateRootProof,
+            _validatorProofs.validatorIndices,
+            _validatorProofs.withdrawalCredentialProofs,
+            _validatorProofs.validatorFields
+        );
+
+        assertEq(stakingNode.unverifiedStakedETH(), _unverifiedStakedETH, "testVerifyWithdrawalCredentials: E0"); // validator already verified withdrawal credentials
+    }
+
+    function testQueueWithdrawal() public {
+        if (!isHolesky) return;
+
+        // uint256 _amount = withdrawalAmount;
+        // fuzzing here will fail on too many inputs rejected
```

Comment:
making fuzzing work with these tests that verifyWithdrawalCredentials may not be feasible.

The amount of logic executed on forked state is very high.

Fuzz testing works well on freshly deployed code that doesn't interact with a ton of pre-existing forked state.

So not surprised by this result.